### PR TITLE
user permission management

### PR DESCRIPTION
**Pull request by jcscottiii**

### CODE REVIEW

File: static/app/controllers.js [hunk 1]
```diff
@@ -49,6 +49,16 @@
         loadOrg(MenuData, $routeParams, $cloudfoundry, $scope);
     });
 
+    app.controller('OrgManagementCtrl', function($scope, $cloudfoundry, $routeParams, MenuData) {
+        loadOrg(MenuData, $routeParams, $cloudfoundry, $scope);
+        var renderOrgUsers = function(users) {
+            console.log(users);
```
Comment (msecret) [inline static/app/controllers.js]: Remove debug statements?

**Comment by jcscottiii** [comment]
image

**Comment by jcscottiii** [comment]
Still working on this

**Comment by jcscottiii** [comment]
image

image

image

Need to switch from using invite by email to invite by username because multiple users can have the same email.

**Pull request merged**